content/observe/hoc.mdx

```diff
@@ -48,3 +48,34 @@ export const Counter = observer<Props>(props => {
   )
 })
 ```
+
+## Babel macro
+
+By default, components wrapped with the observer HOC are anonymous. The `Counter` component from the example above would only
+show up as `wrappedComponent` or `Memo` in the React DevTools or in console log mesages.
+
+For a better developer experience, there is a Babel macro included in the `mobx-react-lite` package, which
+transforms components wrapped with `observer` to retain their name.
+
+In order to use the macro, install [`babel-plugin-macros`](https://github.com/kentcdodds/babel-plugin-macros).
+
+```
```

Comment:
Let's keep documentation on how to configure macros in its respective repo, it doesn't need to be here. Only the last portion on how to use the macro itself is sufficient.